test/FileService.Business.UnitTests/Commands/File/AddNewFileCommandTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using FluentValidation;
+using FluentValidation.Results;
+using LT.DigitalOffice.FileService.Business.Commands.File;
+using LT.DigitalOffice.FileService.Business.Commands.File.Interfaces;
+using LT.DigitalOffice.FileService.Data.Interfaces;
+using LT.DigitalOffice.FileService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.FileService.Models.Db;
+using LT.DigitalOffice.FileService.Models.Dto.Models;
+using LT.DigitalOffice.FileService.Validation.Interfaces;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.FileService.Business.UnitTests.Commands.File
+{
+    public class AddNewFileCommandTests
+    {
```

Comment:
Please, rewrite all Tests names. For example, you rewrite the command name to AddFileCommand, but do not rewrite Test name to AddFileCommandTests

Reply:
Fixed